DUI3-DX/Converters/Autocad/Speckle.Converters.AutocadShared/DependencyInjection/AutofacAutocadConverterModule.cs

```diff
@@ -15,6 +15,7 @@ protected override void Load(ContainerBuilder builder)
     // should be InstancePerLifetimeScope
     // most things should be InstancePerLifetimeScope so we get one per operation
     builder.RegisterType<AutocadConverterToSpeckle>().As<ISpeckleConverterToSpeckle>().SingleInstance();
+    builder.RegisterType<AutocadConverterToHost>().As<ISpeckleConverterToHost>().SingleInstance();
```

Comment:
I suspect the scope should be instance per lifetime

Reply:
Most definitely, considering our last conversation around "unit of work" and scoped factories

Reply:
It is just for `AutocadConverterToHost`? or both `AutocadConverterToHost` and `AutocadConverterToSpeckle`